modules/installation/pages/proxy-setup.adoc

```diff
@@ -229,9 +229,26 @@ If you do not use the reactivation key, you will need to re-register all the cli
 . In the {productname} {webui}, select the newly installed {productname} Proxy, and delete it from the systems list.
 . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
   Click [guimenu]``Generate New Key``, and make a note of the new key.
-. Register the proxy with a bootstrap script as described in xref:installation:proxy-registration.adoc[].
-  In the bootstrap script set the reactivation key with the [systemitem]``ACTIVATION_KEYS`` parameter.
-  For more information about using reactivation keys, see xref:client-configuration:activation-keys.adoc[].
```

Comment:
Actually this is one thing that is missing in the documentation. Compare to https://documentation.suse.com/external-tree/en-us/suma/4.1/suse-manager/installation/proxy-setup.html#replace-susemgrproxy and also https://www.uyuni-project.org/uyuni-docs/uyuni/installation/uyuni-proxy-setup.html

After point 5 you have reactivation key, but it doesn't say what to do with it. So this text should not be removed.

Reply:
I haven't deleted it, it's just moved down after the new section (and is no longer in steps).